src/Microsoft.DotNet.Build.Tasks.Packaging/test.Desktop/project.json

```diff
@@ -8,9 +8,11 @@
     "xunit.runner.visualstudio": "2.1.0"
   },
   "frameworks": {
-    "net45": { }
```

Comment:
After looking, it seems that even though there is a project for Packaging Desktop Tests, looks like the project is empty and has no tests at all, which would explain why no tests are run on the build. In this case we should either remove the project, or if we might add tests in the future then I guess I could add the test-runtime reference to have the setup ready.